<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Aplikasi Penerjemah</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>135</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pilih bahasa sumber :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>175</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pilih bahasa tujuan :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="lang_destination">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>171</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>ar - Arab</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>en - Inggris</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>de - Jerman</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>fr - Perancis</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>it - Itali</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ja - Jepang</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>hi - India</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ko - Korea</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>nl - Belanda</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ru - Rusia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>es - Spanyol</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>th - Thailand</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>tr - Turki</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>vi - Vietnam</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnTerjemahkan">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Terjemahkan!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Teks sumber :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hasil terjemahan:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lang_source">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>129</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
    </property>
    <property name="text">
     <string>id - Indonesia</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtHasilTerjemahan">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSource">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Indonesia ke Bahasa Lainnya</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
